packages/aws-cdk-docs/src/getting-started.rst

```diff
@@ -254,29 +255,30 @@ If needed, compile the code:
 
     .. group-tab:: TypeScript
 
-        To compile your program from **.ts** to **.js**:
+        Use the **npm run build** command to compile your program from **.ts** to **.js**.
 
         .. code-block:: sh
 
             npm run build
 
-        You can also use the **watch** command to continuously compile your code
-        as it changes, so you don't have to invoke the compiler explicitly:
+        You can also use the **npm run watch** command
+	to compiles your code as you save it.

```

Comment:
Indentation

Reply:
* which ~to~ compiles
* also, the fact that it happens continuously is important